internal/plugins/manifests/v2/init.go

```diff
@@ -184,7 +184,7 @@ bundle: manifests kustomize ## Generate bundle manifests and metadata, then vali
 bundle: kustomize ## Generate bundle manifests and metadata, then validate generated files.
 	operator-sdk generate kustomize manifests -q
 	cd config/manager && $(KUSTOMIZE) edit set image controller=$(IMG)
-	$(KUSTOMIZE) build config/manifests | operator-sdk generate bundle -q --overwrite --version $(VERSION) $(BUNDLE_METADATA_OPTS)
+	$(KUSTOMIZE) build config/manifests | operator-sdk generate bundle $(BUNDLE_GEN_FLAGS)
```

Comment:
Can we leave the `--overwrite` flag as is. Removing it will affect the user experience.

Reply:
Ah! missed it! Sounds good!